### PR TITLE
Code comments for timer-modal folder files

### PR DESCRIPTION
**Pull request by kennyt12237**
Added comments for functions, renamed files to follow convention, and more descriptive variable names

Files commented:
- TimerContextProvider.jsx
- timerData.js
- TimerModal.jsx
- TimerSection.jsx

### CODE REVIEW

**Review by franc-hoon — APPROVED** [review]
lgtm

**Comment by IngemarWatt** [comment]
Naming is more explanatory and the commenting is great! Looks good to me

**Review by jeva126 — APPROVED** [review]
Nice work differentiating the work timer from the break timer variables. 👍

File: src/components/current-task/CurrentTask.jsx [hunk 1]
```diff
@@ -24,7 +24,7 @@ function CurrentTask() {
   const [, setPlay] = useContext(PlayContext);
   const [isChecked] = useContext(FullscreenContext);
```
Comment (saaaaraaaah) [inline src/components/current-task/CurrentTask.jsx:25]: isChecked could be slightly more descriptive to not confuse it with a task being checked, but other than that looks good!

**Review by thakurtushar02 — APPROVED** [review]
Nice consistency changes guys

**Pull request merged**